recipe/build.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
+
+set -ex
+
+if [[ "${target_platform}" == linux-* ]]; then
+  CMAKE_ARGS="
+    ${CMAKE_ARGS}
+    -DALSA_FOUND=1
+    -DALSA_INCLUDE_DIRS=${BUILD_PREFIX}/${HOST}/sysroot/usr/include/alsa
+    -DALSA_LDFLAGS=${BUILD_PREFIX}/${HOST}/sysroot/usr/lib64;-lasound
+    -DALSA_LIBRARY_DIRS=${BUILD_PREFIX}/${HOST}/sysroot/usr/lib64
+    -DQT_FEATURE_webengine_ozone_x11=ON
+    -DQT_FEATURE_webengine_system_alsa=ON
+    -DQT_FEATURE_webengine_system_minizip=ON
+  "
+
+  if [[ "${target_platform}" == linux-aarch64 ]]; then
+    # Somehow a path for /usr/include/freetype2 is still making it into the CMake PkgConfig::FREETYPE IMPORT target.
+    # Webenginedriver is just used for tests and does not link against zlib correctly on linux-aarch64.
+    # Chromium vendors minigbm, a discontinued Intel project, which uses compiler features not compatible with our
+    # aarch64 compiler.
+    CMAKE_ARGS="
+      ${CMAKE_ARGS}
+      -DFREETYPE_FOUND=1
+      -DFREETYPE_INCLUDE_DIRS=${PREFIX}/include/freetype2
+      -DFREETYPE_LDFLAGS=${PREFIX}/usr/lib;-lfreetype
+      -DFREETYPE_LIBRARY_DIRS=${PREFIX}/usr/lib
+      -DQT_FEATURE_webengine_system_freetype=OFF
```

Comment:
`QT_FEATURE_webengine_qt_freetype` is off for all platforms, meaning no freetype support in linux-aarch64 I think?

Reply:
I believe the `webengine_qt_*` flags denote whether to use the Qt-vendored dependency and the `webengine_system_*` ones whether or not to use the system, in our case the Conda, dependency. If both are off then it uses the Chromium vendored one.